patch (DOCS-CODE-016): [authN] [sign-in] ensure Azure AD v2.0 usage in the canonical webapp (#6)
Azure-Samples/ms-identity-docs-code-csharp: src/sign-in-webapp/README.md

> ## Scaffold the web app by using the ASP.NET Core Razor project template

Reviewer: New template coming in a few minutes. In the README, they'll learn how to prepare for and set up the sample app (installing packages if they need to) but we won't have them scaffold a _new_ application in the README.

Author: thanks for creating that template @mmacy, we're addressing all readme-related changes from another [pr](#9).

1. Install the latest `Microsoft.Identity.Web` templates from NuGet.org

   ```bash

>    dotnet new --install Microsoft.Identity.Web.ProjectTemplates::1.21.0

Reviewer: ```suggestion
   dotnet new --install Microsoft.Identity.Web.ProjectTemplates
```

But only if you _really_ meant latest. :)

Author: thanks for the feedback @mmacy, we were including that to create the app but there is an explicit request in here to remove that part.

Therefore, we got rid of it from [3959b](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/9/commits/3959bbe9d73c26a2f8db3a7f8777d11575ae3d69) via https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/6#discussion_r761570742

Reviewer 2: Just marking this one resolved to remove the PR block.  @mmacy if there is any more feedback beyond the change that @ferantivero made a week ago, please let us know.

   ```

> 1. Create a Microsoft identity platform (Azure AD v2.0) application. Execute the following command to create the web app project

Reviewer: ```suggestion
1. Create a new web application. Execute the following command to create the web app project:
```

Reviewer 2: Just marking this one resolved to remove the PR block.  @mmacy if there is any more feedback beyond the change that @ferantivero made a week ago, please let us know.

   ```bash

>    dotnet new webapp2 --auth SingleOrg --aad-instance "https://login.microsoftonline.com/" --client-id ${AZURE_AD_APP_CLIENT_ID_WEBAPP} --tenant-id $(az account show --query tenantId --output tsv) --domain ${AZURE_AD_APP_DOMAIN} --called-api-url "https://graph.microsoft.com/v1.0/me"

Reviewer: We should include instructions for only for running the sample application, not creating it.

The tutorials on Docs will walk the customer through adding authentication to a base web app that was generated without using `--auth`.

Author: thanks for the feedback @mmacy, that ^ sounds reasonable. 

Done  | Addressed from [3959b](https://github.com/Azure-Samples/ms-identity-docs-code-csharp/pull/9/commits/3959bbe9d73c26a2f8db3a7f8777d11575ae3d69)

Reviewer 2: Just marking this one resolved to remove the PR block.  @mmacy if there is any more feedback beyond the change that @ferantivero made a week ago, please let us know.

   ```

>    :warning: Today, with ASP.NET Core 3.1 without Microsoft Identity Web, when running `dotnet new --auth` and creating a `webapp` from an ASP.NET core template, the application is targeting the Azure AD v1.0 endpoint. Microsoft Identity Web provides the glue between the ASP.NET Core middleware and MSAL .NET to bring a clearer, more robust developer experience, which also leverages the power of the Microsoft identity platform (formerly Azure AD v2.0 endpoint). `Microsoft.Identity.Web` is a simpler way to use Azure AD in ASP.NET Core web apps and web APIs. It doesn't replace ASP.NET Identity in any way, it doesn't replace AddOpenIdConnect, AddJwtBearer or AddCookie or any of the lower level primitives, but it does use and configure them correctly for Azure AD. It replaces AzureAD.UI and AzureADB2C.UI which are obsolete in .NET 5.0 or later. For more information, please take a look at https://github.com/AzureAD/microsoft-identity-web/wiki/Microsoft-Identity-Web-basics#why-use-microsoftidentityweb

Reviewer: You can cut all this.

```suggestion
```

Reviewer 2: Just marking this one resolved to remove the PR block.  @mmacy if there is any more feedback beyond the change that @ferantivero made a week ago, please let us know.
